client_opts.go

```diff
@@ -121,6 +122,15 @@ func WithPullUnpack(_ *Client, c *RemoteContext) error {
 	return nil
 }
 
+// WithDecryptImageUnpack sets the decryption keys for the client
+func WithDecryptImageUnpack(decryptConfig encconfig.DecryptConfig) RemoteOpt {
```

Comment:
These opts should live in the encryption package

Reply:
Looks like this now:
```
// WithDecryptImageUnpack sets the decryption keys for the client
func WithDecryptImageUnpack(unpackOpt UnpackOpt) RemoteOpt {
        return func(_ *Client, c *RemoteContext) error {
                c.UnpackOpts = append(c.UnpackOpts, unpackOpt)
                c.Unpack = true
                return nil
        }
}
```
Should be generic enough to keep it there?